aiida_quantumespresso/parsers/pp.py

```diff
@@ -92,6 +92,16 @@ def parse(self, **kwargs):
             self.logger.error(traceback.format_exc())
             return self.exit_codes.ERROR_UNEXPECTED_PARSER_EXCEPTION
 
+        if 'plot_num' not in self.output_parameters:
+            try:
+                stdref_raw = retrieved.get_object_content('aiida.ref')
+                for dataline in stdref_raw.splitlines():
+                    if 'plot_num' in dataline:
+                        self.output_parameters['plot_num'] = int(dataline.split('=')[1])
+            except Exception:
+                self.logger.error(traceback.format_exc())
+                return self.exit_codes.ERROR_UNEXPECTED_PARSER_EXCEPTION
+
```

Comment:
This is why I need the `aiida.ref` file, if I'm doing just the second step, `plot_num` does not appear in the output and thus won't be parsed in the section above this one.

The whole PR in general, but specially this parsing of the `plot_num` is very "tied together with wire", as we say (precariously done, with many specifics such as hardcoding the `aiida.ref` filename, etc), just to illustrate what would need to be modified. Ideas about how to properly do this are welcome.